### ISSUE TITLE
Omit sensitive breaches from the "Latest Breach Added" section on the homepage

### ISSUE THREAD

**Issue by changecourse** [mozilla/blurts-server#1321]
Using the boolean .IsSensitive flag via HIBP to prevent sensitive breach cards from appearing on our home page, even if a sensitive breach is technically the latest breach in the db.

**Comment by pdehaan** [comment]
I posted this in [Slack](https://mozilla.slack.com/archives/C93K0MBM5/p1572379295384200), but might work here as well...

https://github.com/mozilla/blurts-server/blob/d99aa8133628202a15ce2471869bf5e3714ed04f/hibp.js#L153-L164

Not sure how many places use that `getLatestBreach()` method, and if it's just possible to add a Boolean `ignoreSensitive` flag (default `false` so we don't change existing behavior) and then just pass a `true` arg to the homepage template.

```sh
git grep -n "getLatestBreach("

hibp.js:94:      app.locals.latestBreach = this.getLatestBreach(breaches);
hibp.js:153:  getLatestBreach(breaches) {
tests/hibp.test.js:39:  const latestBreach = hibp.getLatestBreach(testBreaches);
```

---

Also (and unrelated) but why loop over all breaches and do a compare on each iteration vs just sorting the breach array and plucking the first/last value? If we sorted by AddedDate when importing from HIBP, we might only have to perform the  search once and cut down on a bit of looping. 🤷‍♂
